components/FriendsCounter.js

```diff
@@ -0,0 +1,36 @@
+import styled from "styled-components";
+
+function getFriendIndex(data) {
+  return data.length;
+}
+
+const StyledfriendsCounter = styled.div`
+  display: flex;
+  flex-direction: row;
```

Comment:
`flex-direction: fow` is the default value and does not need to be set. Maybe you do not even need flexbox here if you follow my other suggestions.